services/fortuneo-auth/fortuneo_parser.py

```
@@ -0,0 +1,1043 @@
+def _normalize_spaces(value: str) -> str:
+    """Fold every Unicode space separator to a plain space.
+
+    Fortuneo groups thousands with a space, but not always the same one: a
+    regular space, U+00A0 and U+202F have all been seen, and a securities
+    history page also uses U+2009. Handling them one at a time is how this
+    parser has already failed twice, silently, on pages that did hold the data
+    -- so every `Zs` character is folded, plus the fixed-width spaces that are
+    not in that category.
+    """
+    return "".join(
+        " " if (unicodedata.category(char) == "Zs" or char in "  ") else char
```

Ruff (0.16.2)

[warning] String contains ambiguous ` ` (FIGURE SPACE). Did you mean ` ` (SPACE)?

(RUF001)